src/Tribe/Main.php

```diff
@@ -1474,7 +1474,7 @@ public function get_event_label_singular() {
 		/**
 		 * Load the day view template tags
 		 * Loaded late due to potential upgrade conflict since moving them from pro
-		 * @TODO move this require to be with the rest of the template tag includes in 3.9
+		 * @todo move this require to be with the rest of the template tag includes in 3.9
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: 73701ec576209680ac7b0f988ac7d0fb</sub>